frontend/libs/studio-content-library/src/ContentLibrary/LibraryBody/pages/CodeListPage/CodeLists/CodeLists.tsx

```diff
@@ -133,3 +129,16 @@ function CodeListAccordionContent({
     </Accordion.Content>
   );
 }
+
+function InvalidCodeListAlert(): React.ReactElement {
+  return (
+    <StudioAlert size='small' severity='danger'>
+      <StudioParagraph size='small'>
```

Comment:
Can this be a `span` instead? Then we avoid depending on the `StudioParagraph` when we do not need it 🫣